src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -2,21 +2,50 @@
 
 import java.util.Optional;
 import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 
 public class ActorDaoImpl extends AbstractDao implements ActorDao {
+    private static final String FAILED_ADDING_ACTOR
```

Comment:
```suggestion
    private static final String FAILED_ADDING_ACTOR_MESSAGE
```
fix everywhere

but commonly we don't create constants out of exception messages